sdk/identity/identity/test/node/msalNodeTestSetup.ts

```diff
@@ -0,0 +1,229 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+import Sinon, { createSandbox } from "sinon";
+import { MsalBaseUtilities } from "../../src/msal/utils";
+import { Recorder } from "@azure-tools/test-recorder";
```

Comment:
To test what Jeff said, you can try importing dotenv from "@azure-tools/test-recorder"

Reply:
Done!